torch/onnx/symbolic_opset9.py

```diff
@@ -941,6 +941,8 @@ def expand_as(g: jit_utils.GraphContext, self, other):
             if torch.equal(self_t.mean(d).unsqueeze(d).expand_as(self_t), self_t):
                 dims.append(d)
                 self = g.op("Constant", value_t=self_t.mean(dims).to(orig_type))
```

Comment:
```suggestion
                self = g.op("Constant", value_t=self_t.mean(dims, keepdims=True).to(orig_type))
```

Looking at this again, I think this is the right fix.

Reply:
Yeah, thanks a lot! Updated.